src/writer.js

```diff
@@ -0,0 +1,377 @@
+//create html element with given attributes
+export function createHtml (selector, className, innerText, ...keysValues) {
+  let elem = document.createElement(selector);
+  className == undefined ? elem.className = '' : elem.className = className;
+  innerText == undefined ? elem.innerHTML = '' : elem.innerHTML = innerText;
+  if (keysValues.length != 0){
+    let i = 1;
+    while (i < keysValues.length) {
+      elem.setAttribute(keysValues[i-1], keysValues[i]);
+      i += 2;
+    }
+  }
+  return elem;
+};
+
+var classArrLength = 0;
+export function addCards (num) {
+  let classArray = ['card1', 'card1', 'card2', 'card2', 'card3', 'card3', 'card4', 'card4', 'card5', 'card5', 'card6', 'card6', 'card7', 'card7', 'card8', 'card8', 'card9', 'card9', 'card10', 'card10', 'card11', 'card11', 'card12', 'card12', 'card13', 'card13', 'card14', 'card14', 'card15', 'card15', 'card16', 'card16', 'card17', 'card17', 'card18', 'card18'];
```

Comment:
you got tired when you wrote this array by hands, aren't you? why you cannot just use a cycle, like

```
for(let i=0; i< 10; i++) { classArray.push(`card${i}`,`card${i}`);
```

Reply:
Done